docs/Rake-Task.md

```diff
@@ -33,7 +33,7 @@ An more sophisticated rake task that would make use of all available configurati
 ```Ruby
 Reek::Rake::Task.new do |t|
   t.name          = 'custom_rake' # Whatever name you want. Defaults to "reek".
-  t.config_file   = 'config/.reek.yml' # Defaults to nothing.
+  t.config_file   = 'config/config.reek' # Defaults to nothing.
```

Comment:
This can stay as it is I think.

Reply:
While it's true that we're not checking for the file extension explicitly when specifying a config file I still think it's very confusing that all of a sudden we have a configuration file with a yaml extension.

Reply:
Oh wait I missed that this was also in the docs. Please ignore my remark for this line.